streams/src/main/java/org/apache/kafka/streams/kstream/Suppressed.java

```diff
@@ -118,6 +119,25 @@ static StrictBufferConfig unbounded() {
          * duplicate results downstream, but does not promise to eliminate them.
          */
         EagerBufferConfig emitEarlyWhenFull();
+
+        /**
+         * Disable the changelog for store built by this {@link StoreBuilder}.
+         * This will turn off fault-tolerance for your store.
```

Comment:
```suggestion
         * This will turn off fault-tolerance for the suppression, and will result in data loss in the event of a rebalance.
```

This isn't a "store", but rather a buffer internally used for suppression. Also, it seems appropriate to be a little more dire in our warning here because the internal nature of the buffer may make it less obvious to people what the downside of disabling this changelog is.